tests/test_core.py

```diff
@@ -4649,73 +4578,47 @@ def test_sscanf_whitespace(self):
 
     self.do_run_from_file(src, output)
 
-  @no_wasm_backend
+  # @no_wasm_backend
```

Comment:
Whoops, that was a temporary test thing. I seem to have correctly uncommented it in a future commit though - latest has it https://github.com/jgravelle-google/emscripten/blob/c6c84c92521f01ce1a5bc76db54db79a9b074e4b/tests/test_core.py#L4203